src/apps/explorer/pages/Solver/data.ts

```diff
@@ -0,0 +1,106 @@
+export const ACTIVE_SOLVERS = {
```

Comment:
I've created a way to fetch this info dynamically from Dune/Github https://github.com/cowprotocol/explorer/pull/226